assets/src/infrastructure/ui/display/Alert/Error.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Alert, AlertIcon, AlertTitle, AlertDescription } from '@chakra-ui/core';
+
+interface ErrorProps {
+	description: string;
+	title: string;
+}
+
+const Error: React.FC<ErrorProps> = ({ description, title }) => {
+	return (
+		<Alert
+			status='error'
```

Comment:
Had a comment written out about how the `EmptyState` and `ErrorIndicator` components should not be in the infrastructure layer but you've gone and changed things so it's not applicable anymore

### 😆 

This `Error` component you have now is better but could still be improved.

This is still a bit too specific for an adapter because it is applying a somewhat specific use case to the underlying Chakra `Alert` component (ie: an error)

So what you need to do is create a single adapter for the  `Alert` component that can be used by any component in the app layer that needs to display some sort of static notification (as opposed to a popup notice). Then reinstate the `EmptyState`, `ErrorIndicator`, and any other components in the app layer that need this, and have those use the new adapter instead of importing from Chakra directly.

The new adapter for the Chakra `Alert` component could be named something like:

- Notification

- Notice

- Banner (meh)

- Alert (although this sounds more like it's just for warnings)

 Then we can also create other more specific components in the app layer for things like:

- success notice

- error notice

- info notice

- warning notice

that would simply apply a bit of extra configuration to the new adapter component. Client code in specific use cases can then utilize these components in the app layer, but only have to add a few extra details, like the actual text to be displayed for the notice as well as set some other style related props like "notice size" etc

Reply:
named `Alert` for simplicity: https://github.com/eventespresso/event-espresso-core/pull/2682/commits/04660d043aca0de7fc59ac25f10c87a5e76cd9c4
***
the `ErrorMessage` which is consuming it has a fairly limited API surface:

```
...
	if (asAlert) {
		return <Alert description={errorMessage} status='error' title={__('Error')} />;
	}
...
```